### ISSUE TITLE
Problem with Godot 2.1 stable

### ISSUE THREAD

**Issue by ShinZ6** [Mavhod/GodotAdmob#5]
After compiling with 2.1 stable source code, when exporting to android if the admob module is enabled the game crashes on the phone.

**Comment by Mavhod** [comment]
Now I can't compile with Andorid it error about virtual script, I'll check it later when that error is fixed.

**Comment by ShinZ6** [comment]
Don't compile with the latest source code, that's godot 2.2 alpha, you should compile the source code of godot 2.1 stable in the release section.

**Comment by Mavhod** [comment]
I load .zip from github with branch 2.1 after build it and test It work fine, Is you mistake something?

There is template I was build it, you can download for test:
https://www.dropbox.com/s/juaz9vvzy5aumlm/androidTemplate2.1.apk?dl=0

**Comment by rochmaddn** [comment]
godot 2.0.4.1 admob example crash on my phone
help me please.

**Comment by ShinZ6** [comment]
Thanks @Mavhod  works fine with the template you gave me, how can I build it by myself next time, maybe with scons platform=android ?

**Comment by Mavhod** [comment]
Yes, I use `scons platform=android target=release`, you can follow this instruction
http://docs.godotengine.org/en/latest/reference/compiling_for_android.html

rochfe, I was compiled version 2.0 and it work fine, I don't know what are you wrong.

**Comment by Nearoo** [comment]
Remember, you can always use `adb logcat -s godot` to see what's going on on your phone....
